### ISSUE TITLE
opbg_tunnel3 lightcomponent sprite overlapping

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#12769]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Sprites are overlapping in Tunnelwalls.png, as part of the lightcomponent of opbg_tunnel3

![image](https://github.com/Regalis11/Barotrauma/assets/73229309/21271cb6-681b-40f8-9ea1-5d2b9e62f9e6)

![image](https://github.com/Regalis11/Barotrauma/assets/73229309/e10d935c-736f-46aa-8174-12ba62c20ec0)

In OutpostItems.xml

```xml
  <Item name="" identifier="opbg_tunnel3" allowattachitems="true" width="831" height="416" scale="0.5" category="Decorative" noninteractable="true">
    <sprite texture="Content/Map/Outposts/Art/TunnelWalls.png" sourcerect="0,416,831,416" depth="0.98" premultiplyalpha="false" origin="0.5,0.5" />
    <LightComponent range="0.0" lightcolor="255,56,56,111" IsOn="true" castshadows="false" allowingameediting="false" flicker="0.1" flickerspeed="0.1" blinkfrequency="1">      
      <Upgrade gameversion="0.17.15.0" range="0" />
      <sprite texture="Content/Map/Outposts/Art/TunnelWalls.png" sourcerect="0,1248,832,416" depth="0.1" origin="0.5,0.5" alpha="1.0" />
    </LightComponent>
  </Item>
```

### Version

v1.0.21.0

**Comment by NamAnhLe** [comment]
Fix pushed in 01a0cb0

**Comment by Regalis11** [comment]
Tested, couldn't spot any issues
